catbuffer/schemas/nem/block.cats

```diff
@@ -16,7 +16,7 @@ struct Block
 	inline EntityBody
 
 	# previous block hash
-	previous_block_hash = Hash256
+	previous_block_hash = inline SizePrefixedHash256
```

Comment:
isn't there an extra size somewhere? 🤔

Reply:
yeah, you are right.  I forgot to add it. 😅